analytics/client.go

```diff
@@ -271,6 +272,10 @@ func (c *client) sendRawEventRequest(method string, path string, body string) er
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != http.StatusOK {
+		fmt.Printf("\nRequest response error (%d): %s\n", resp.StatusCode, string(body))
```

Comment:
Will replace by just generating and returning an error with the body of the response and the response status.